CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+### Repo Structure
+```
+# Entity Definitions
+domain/                    # This is the core directory where the domains are defined.
+  abstracts/               # Abstract base classes for common attributes and methods.
+  models/                  # Domain interface definitions, e.g. model, assistant.
+  repositories/            # Extensions abstract and interface
+
+# Business Rules
+usecases/                  # Application logic 
+	assistants/              # CRUD logic (invokes dtos, entities).
+	chat/                    # Logic for chat functionalities.
+	models/                  # Logic for model operations.
+
+# Adapters & Implementations
+infrastructure/            # Implementations for Cortex interactions
+  commanders/              # CLI handlers
+    models/
+    questions/             # CLI installation UX
+    shortcuts/             # CLI chained syntax
+    types/
+    usecases/              # Invokes UseCases
+
+  controllers/             # Nest controllers and HTTP routes
+		assistants/						 # Invokes UseCases
+	  chat/     						 # Invokes UseCases
+		models/                # Invokes UseCases
```

Comment:
Is it a GitHub preview format issue or an indentation issue? @@

Reply:
hmm weird, on my local seems fine, will update this